phpstan.dist.neon

```diff
@@ -16,6 +16,7 @@ parameters:
     ignoreErrors:
         - '/^Class [a-zA-Z\\]+ extends @final class PhpCsFixer\\(ConfigurationException\\InvalidConfigurationException|ConfigurationException\\InvalidFixerConfigurationException|Tokenizer\\Tokens)\.$/'
         - '/^\$this\(PhpCsFixer\\Tokenizer\\Tokens\) does not accept PhpCsFixer\\Tokenizer\\Token\|null\.$/'
+        - '/^Class PhpCsFixer\\Console\\Command\\CheckCommand extends @final class PhpCsFixer\\Console\\Command\\FixCommand\.$/'
```

Comment:
ouch

Reply:
Yeah, not the best thing, but without `@final` annotation tests fail. So in order to use inheritance approach we either need to ignore this in PHPStan or exclude `FixCommand` from test which enforces classes being `final` 🤷‍♂️.

Reply:
I think we could refactor this later, while keeping 100% backward compatibility for `check` command. But people could already use it as a quick-win solution.

BTW. same thing is already ignored 2 lines above, just realised this 😅. I could just improve the regex.